.github/workflows/ci.yaml

```diff
@@ -42,7 +42,9 @@ jobs:
 
   build:
     name: Build charms
-    uses: canonical/data-platform-workflows/.github/workflows/build_charms_with_cache.yaml@v2
+    uses: canonical/data-platform-workflows/.github/workflows/build_charms_with_cache.yaml@v8.0.0
```

Comment:
We have already version 11 of shared workflows... but we need to have here renovate anyway... will be updated later. LGTM.